core/CronArchive.php

```diff
@@ -978,10 +977,10 @@ protected function processArchiveDays($idSite, $lastTimestampWebsiteProcessedDay
     public function isThereAValidArchiveForPeriod($idSite, $period, $date, $segment = '')
     {
         if (Range::isMultiplePeriod($date, $period)) {
-            $rangePeriod = Factory::build($period, $date, Site::getTimezoneFor($idSite));
+            $rangePeriod = PeriodFactory::build($period, $date, Site::getTimezoneFor($idSite));
```

Comment:
@sgiehl is that change needed or is it actually causing an issue? Just asking out of curiosity.

Reply:
Not really needed. But it's a bit messy to import the same class twice with different aliases...